### PR TITLE
fix(react-chart): update ZoomAndPan plugin ts declaration

### PR DESCRIPTION
**Pull request by DmitryBogomolov**
Fixes https://github.com/DevExpress/devextreme-reactive/issues/2107.

Renames *packages/dx-react-chart-<theme>/src/plugins/zoom-pan.jsx* files so that they are not missed by the **ts-generator**.
Renames `ViewportOptions` to `Viewport` so that it is synchronized with entities in *md* files.

### CODE REVIEW

File: packages/dx-react-chart/docs/reference/zoom-and-pan.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ import { ZoomAndPan } from '@devexpress/dx-react-chart';
...
+onViewportChange? | (viewport: [Viewport](#viewport)) => void | A function that is executed when the viewport changes.
 interactionWithArguments? | 'none' &#124; 'pan' &#124; 'zoom' &#124; 'both' | The type of interaction available for the argument scale.
 interactionWithValues? | 'none' &#124; 'pan' &#124; 'zoom' &#124; 'both' | The type of interaction available for the value scale.
 zoomRegionKey? | 'shift' &#124; 'alt' &#124; 'ctrl' | The key the allows a user to zoom a region by selecting it with the mouse drag gesture.
 dragBoxComponent? | ComponentType&lt;[ZoomAndPan.DragBoxProps](#zoomandpandragboxprops)&gt; | A component that renders the rectangle that appears when a user zooms a region.
 
 ## Interfaces
 
-### ZoomAndPan.Viewport
+### Viewport
```
Comment (Krijovnick) [inline packages/dx-react-chart/docs/reference/zoom-and-pan.md]: In most files we use the rule `Plugin_Name.Interface_Name` for declaring interfaces.

File: packages/dx-react-chart-material-ui/src/index.js [hunk 2]
```diff
@@ -13,4 +13,4 @@ export { ValueAxis } from './plugins/value-axis';
 export { ArgumentAxis } from './plugins/argument-axis';
 
 export { Tooltip } from './plugins/tooltip';
-export { ZoomAndPan } from './plugins/zoom-pan';
+export { ZoomAndPan } from './plugins/zoom-and-pan';
```
Comment (Krijovnick) [inline packages/dx-react-chart-material-ui/src/index.js]: Should we rename the folder name `zoom-pan` in the `dx-chart-core`?

File: packages/dx-react-chart/docs/reference/zoom-and-pan.md (hunk 1 above)
Comment (DmitryBogomolov) [reply packages/dx-react-chart/docs/reference/zoom-and-pan.md]: `PluginName` prefix affects generated ts code. The following is generated

```ts
export namespace ZoomAndPan {
  /** The viewport configuration. */
  export type Viewport = ZoomAndPanBase.Viewport;
}
```

and causes an error as there is no `Viewport` field in `ZoomAndPanBase`.

File: packages/dx-react-chart-material-ui/src/index.js (hunk 2 above)
Comment (DmitryBogomolov) [reply packages/dx-react-chart-material-ui/src/index.js]: Yes, we should.

Comment (DmitryBogomolov) [reply packages/dx-react-chart-material-ui/src/index.js]: Renamed.

**Pull request merged**